### PR TITLE
Show staked balance

### PR DESCRIPTION
**Pull request by abhay**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ anyhow = "1.0"
 thiserror = "1"
 byteorder = "1"
 structopt = "0"
-helium-api = "3"
+helium-api = { git = "https://github.com/helium/helium-api-rs", branch="ak/staked-balance"}
```
Comment (abhay) [inline Cargo.toml:13]: WIP

**Comment by lthiery** [comment]
Waiting for https://github.com/helium/helium-wallet-rs/pull/226

File: src/txns/balance.rs [hunk 2]
```diff
@@ -47,24 +47,36 @@ async fn print_balance(pubkeys: &[PublicKey]) -> Result {
     // sample the first pubkey to determine network
     let network = pubkeys[0].network;
 
-    let client = Client::new_with_base_url(api_url(network));
```
Comment (lthiery) [inline src/txns/balance.rs:50]: Oddly, this is not a diff with master...

Comment (lthiery) [reply src/txns/balance.rs:50]: I guess it's probably because the PR had this diff with the original master? Weird

**Pull request merged**